sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -15,8 +15,8 @@
  */
 public final class DirectConnectionConfig {
     //  Constants
-    private static final Duration DEFAULT_IDLE_ENDPOINT_TIMEOUT = Duration.ofSeconds(70L);
-    private static final Duration DEFAULT_CONNECT_TIMEOUT = Duration.ofSeconds(60L);
+    private static final Duration DEFAULT_IDLE_ENDPOINT_TIMEOUT = Duration.ofHours(24L);
```

Comment:
Isn't this too much ?
For big customers, who have very high number of physical partitions - like around 500 or 1000. Then this will always create that many connections -> physical partitions * 4 (which is the number of replicas) - and will keep the resources occupied for 24 hours even if we don't need them.

Reply:
see the .Net default:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.client.connectionpolicy.idletcpconnectiontimeout?view=azure-dotnet#Microsoft_Azure_Documents_Client_ConnectionPolicy_IdleTcpConnectionTimeout

having 70 seconds as timeout certainly cause latency issue. and a big customer with those many partitions certainly wants a good latency.

We can think about what the good default should be 70 seconds is too low.

Reply:
Agreed, 70 seconds is too low from what we have seen so far. 
But In my opinion, 24 hours is a lot. It will just occupy many resources that we don't want the code to retain if they are not being used. 
May be start with 1 hour ?
@kirankumarkolli @David-Noble-at-work thoughts ?

Reply:
1h seems a good start. changed to 1h.

Reply:
One single configuration value doesn't fit for all CX needs.

My first thoughts are we will have few top usage profiles (covering some trade-off's) which gives the CX the easier way to pick and choose. @David-Noble-at-work is exploring those opitions. 

Till then parity with .NET might be reasonable middle ground. 

Also pinged Hemant offline for his opinion.